client-admin/src/components/conversations-and-account/conversation.js

```diff
@@ -0,0 +1,13 @@
+/** @jsx jsx */
+import { jsx, Text, Card } from "theme-ui";
+
+export default ({ c, i, goToConversation }) => {
+  return (
+    <Card onClick={goToConversation} sx={{ cursor: "pointer", mb: [3] }} key={i}>
```

Comment:
```suggestion
    <Card onClick={goToConversation} sx={{ cursor: "pointer", overflow: "hidden", mb: [3] }} key={i}>
```

Accepting change fixed overflow like so, which most commonly shows up when long urls are used in markdown of description:
<img width="618" alt="Screen Shot 2020-06-07 at 11 22 41 PM" src="https://user-images.githubusercontent.com/305339/83987279-1088d900-a916-11ea-84bf-d56be29094df.png">

Reply:
Why not https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap ?

Reply:
Perf! Didn't know about it. Will add after I check how to apply it

Reply:
@colinmegill Confirmed this works well! Thanks. Good to accept change?

<img width="565" alt="Screen Shot 2020-06-09 at 3 43 37 PM" src="https://user-images.githubusercontent.com/305339/84187172-0a0a7680-aa68-11ea-8ccd-57ec657490bd.png">